src/foam/nanos/auth/Permission.js

```diff
@@ -23,17 +23,17 @@
    ],
 
    methods: [
-     function implies(permission) {
+     function implies(permissionId) {
        if ( this.id.endsWith('*') ) {
          var prefix = this.id.substring(0, this.id.length-1);
 
          // Check that is a valid permission string (should be done as a property validator)
          if ( prefix.length && ! prefix.endsWith('.') ) return false;
 
-         return permission.startsWith(prefix);
+         return permissionId.startsWith(prefix);
        }
 
-       return permission == this.id;
+       return false;
```

Comment:
This line needs to be return permissionId == this.id; in case the permission is an exact match.